packages/client/server.js

```diff
@@ -1,36 +1,36 @@
-const path = require('path');
-const fs = require('fs');
-const express = require('express');
-const expressStaticGzip = require('express-static-gzip');
-const packageRoot = require('app-root-path').path;
-const https = require('https');
-const http = require('http');
+import { join } from 'path';
+import { readFileSync } from 'fs';
+import Koa from '@feathersjs/koa';
+import serve from 'koa-static';
+import { path as packageRoot } from 'app-root-path';
+import { createServer } from 'https';
+import { createServer as _createServer } from 'http';
 
-
-const app = express();
+const app = new Koa();
 const PORT = process.env.HOST_PORT || 3000;
-const HTTPS = process.env.VITE_LOCAL_BUILD ?? false
+const HTTPS = process.env.VITE_LOCAL_BUILD ?? false;
 
-app.use(expressStaticGzip(path.join(packageRoot, 'packages', 'client', 'dist'), {
-  enableBrotli: true,
-  serveStatic: {
-    setHeaders: (res) => {
-      res.set('Origin-Agent-Cluster', '?1')
-    }
+app.use(serve(join(packageRoot, 'packages', 'client', 'dist'), {
+  brotli: true,
+  setHeaders: (res) => {
+    res.set('Origin-Agent-Cluster', '?1')
```

Comment:
I believe that this needs to be cts.setHeader('Origin-Agent-Cluster', '?1') (and change the parameter (res) to (ctx)).